### DISCUSSION TITLE
Customize "more" text

### DISCUSSION THREAD

**Discussion by Kistelekig98** [gismofx/toast_ui.blazor_calendar#26] [Ideas]
It would be great, if I could customize the "more" text in month view. I can set the daynames to hungarian, but the "more" remains english yet :)

**Comment by gismofx** [comment]
@Kistelekig98 Thanks. I'm happy to help implement this. 

If you have some spare time, it would be helpful if you could try to search the tui calendar repo and documents site to see if they expose that capability. If nothing found we could ask the question on the repo.

**Reply by gismofx** [reply]
I posted the question here: 
https://github.com/nhn/tui.calendar/issues/1066

**Reply by gismofx** [reply]
Looks like we can change it. I need to implement this month template. Stay tuned.

**Reply by gismofx** [reply]
Ref: https://nhn.github.io/tui.calendar/latest/tutorial-example14-template-for-month

**Comment by gismofx** [comment]
@Kistelekig98 It looks like the customization happens in JS, by overriding the existing methods. I need to figure out how to implement this from C#. I'm open to ideas.

**Reply by Kistelekig98** [reply]
Thank you for your quick response! :)

**Comment by gismofx** [comment]
@Kistelekig98 Making a little progress on this one:
![image](https://user-images.githubusercontent.com/6783791/162646193-9d9243ad-1a0c-4e52-9547-959219770bcf.png)

In `InitCalendar` you add something like this:
```csharp
var calendarTemplate = new TUITemplate();
calendarTemplate.AddMonthGridHeaderExceed(@"return '<span class=""weekday-grid-more-schedules"">+' + hiddenSchedules + ' says gismofx' + '</span>';");
```

These customizable functions are coming here:
https://nhn.github.io/tui.calendar/latest/Template

Lots more functions need to be added for full customization. Some customization can be somewhat complex and I don't have a C# solution yet. Maybe it's not necessary and it may be that you have the template function call into some JS code that the user has to write. 

I pushed the updates to the DEV branch if you want to see and if you want to help create some of the functions for the customization that you want. I'd happily accept a pull request.

**Comment by gismofx** [comment]
@Kistelekig98 I've pushed this customization for "more" into the latest pre-release 1.0.0-beta3.0 on nuget.
